CMakeLists.txt

```diff
@@ -53,10 +53,9 @@ find_package(Foundation QUIET)
 find_package(SQLite3 REQUIRED)
 
 # Enable `package` modifier for the whole package.
+add_compile_options("$<$<COMPILE_LANGUAGE:Swift>:-package-name;SwiftPM>")
```

Comment:
Do we want to `$<$<COMPILE_LANGUAGE:Swift>:SHELL:-package-name SwiftPM>` the generator expression in case someone somewhere else tries to throw in a `-package-name <blah>`? Don't want CMake de-duping either part of this.

Reply:
I think that is a good idea, but we should do that as a follow up (I will be renaming the package name, I can throw it into that change).